src/components/EntityPage/Participant/index.js

```diff
@@ -53,6 +52,11 @@ class ParticipantEntity extends React.Component {
   render() {
     const { participantId, location, participant, isLoading, error } = this.props;
 
+    //only way to update the page when we click a link to a participant in clinical...
+    if(this.props.participant !== null && this.props.participant.kf_id !== this.props.participantId) {
```

Comment:
Should you check isLoading too to prevent sending the query many times?

Reply:
From what I've tested the check to see if this.props.participant is null handles that, but I can add a check to isLoading to be sure!